order/management/tests/test_delete_dummy_orders.py

```diff
@@ -0,0 +1,25 @@
+"""Unit tests for Delete dummy orders command."""
+
+from io import StringIO
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from order.models import Order
+from tms.constants import DUMMY_EMAIL_MARKER, DUMMY_ORDER_MARKER
+from tms.utils import TestDbSetUp
+
+
+class DeleteDummyOrdersTest(TestCase, TestDbSetUp):
+    """Dummy orders deletion test class."""
+
+    def setUp(self):
+        """Setup order for deletion."""
+        self.client = self.create_client(email_post_fix=DUMMY_EMAIL_MARKER)
+
+    def test_delete_dummy_orders(self):
+        """Test delete dummy orders command."""
```

Comment:
for all delete tests. add a new line 
self.assertTrue(Order.objects.filter(order=DUMMY_ORDER_MARKER).exists())
as the first line to make sure before we run the command dummy data exists which we are going to delete.